scripts/lint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+docker run \
+  -e RUN_LOCAL=true \
```

Comment:
In dev.sh in function format(), can we now call lint.sh instead of dotnet-format?